drivers/network/dd/e1000/hardware.c

```diff
@@ -189,32 +190,74 @@ static BOOLEAN E1000ReadMdic(IN PE1000_ADAPTER Adapter, IN ULONG Address, USHORT
 }
 #endif
 
-
 static BOOLEAN E1000ReadEeprom(IN PE1000_ADAPTER Adapter, IN UCHAR Address, USHORT *Result)
 {
     ULONG Value;
     UINT n;
 
-    E1000WriteUlong(Adapter, E1000_REG_EERD, E1000_EERD_START | ((UINT)Address << E1000_EERD_ADDR_SHIFT));
+    if (Adapter->E1000_EERD_ADDR_SHIFT == 0xffff) {
```

Comment:
Keep with the style of the file please: braces on their own line.

Reply:
Done